src/05-led-roulette/build-it.md

```diff
@@ -30,48 +30,53 @@ With the `rust-std` component in place you can now cross compile the program usi
 ``` console
 $ # make sure you are in the `src/05-led-roulette` directory
 
-$ cargo build --target thumbv7em-none-eabihf
+$ cargo build --target thumbv6m-none-eabi
    Compiling semver-parser v0.7.0
-   Compiling aligned v0.1.1
-   Compiling libc v0.2.35
-   Compiling bare-metal v0.1.1
-   Compiling cast v0.2.2
-   Compiling cortex-m v0.4.3
-   (..)
-   Compiling stm32f30x v0.6.0
-   Compiling stm32f30x-hal v0.1.2
-   Compiling aux5 v0.1.0 (file://$PWD/aux)
-   Compiling led-roulette v0.1.0 (file://$PWD)
-    Finished dev [unoptimized + debuginfo] target(s) in 35.84 secs
+   Compiling typenum v1.12.0
+   Compiling proc-macro2 v1.0.19
+   Compiling unicode-xid v0.2.1
+   Compiling cortex-m v0.6.3
+   (...)
+   Compiling as-slice v0.1.3
+   Compiling aligned v0.3.4
+   Compiling cortex-m-rt-macros v0.1.8
+   Compiling nrf-hal-common v0.11.1
+    Finished dev [unoptimized + debuginfo] target(s) in 18.69s
 ```
 
-> **NOTE** Be sure to compile this crate *without* optimizations. The provided Cargo.toml file and build command above will ensure optimizations are off. 
```

Comment:
The note about optimizations is still useful I think. Otherwise people will not know what they have to do/if they have to do something else.